WeArtClient.cpp

```diff
@@ -9,11 +9,22 @@
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #include <ws2tcpip.h>
-#include <windows.h>
-
 
 #include <ppltasks.h>
-using namespace Windows::Foundation;
+
+#include <roapi.h>
+#include <wrl.h>
+
+#include <Windows.Foundation.h>
+#include <Windows.System.Threading.h>
+#include <wrl/event.h>
+#include <stdio.h>
+#include <Objbase.h>
+
+using namespace ABI::Windows::Foundation;
+using namespace ABI::Windows::System::Threading;
+using namespace Microsoft::WRL;
+using namespace Microsoft::WRL::Wrappers;
```

Comment:
It is not clear how this changes are related to the CMake build system, and I guess upstream may wonder the same.

Reply:
I need to investigate further this. This is indeed the trickiest point

Reply:
I have started simplifying the code avoid unnecessary includes and ``using namespace``. I will now add a guard to divide the two compilation procedures

Reply:
I have added the guard